rbac/management/group/view.py

```diff
@@ -796,10 +800,47 @@ def _list_principals_in_group(self, request: Request, uuid: Optional[UUID] = Non
         if isinstance(resp, dict) and "errors" in resp:
             return Response(status=resp.get("status_code"), data=resp.get("errors"))
 
-        page = self.paginate_queryset(resp.get("data"))
-        response = self.get_paginated_response(page)
-
-        return response
+        path = request.path
```

Comment:
it looks like the response from:
```
resp = proxy.request_filtered_principals(username_list, org_id=org_id, options=options)
```
is not in the expected structure, currently we return from [here](https://github.com/RedHatInsights/insights-rbac/blob/03729ea52d3b86b308ba2b8fd12d2cc33820d6df/rbac/management/principal/proxy.py#L170)
```
data = {"userCount": len(userList), "users": paginatedUserList}
return dict(data=data, status_code=200)
```

however the expected structure seems to be
```
{
    "data": [
        {"username": "test1"},
        {"username": "test2"}
    ],
    "status_code": 200,
    "userCount": 2,
}
```
I believe this could be an option how to align the return value with expected format, what do you think?